lib/themes/dosomething/paraneue_dosomething/js/reportback/templates/reportback.tpl.html

```diff
@@ -1,10 +1,21 @@
 <% if (isListItem) { %><li><% } %>
-<div class="photo -stacked -framed" data-reportback-item-id="...">
+<div class="photo -stacked -framed" data-reportback-item-id="<%= id %>">
   <figure class="wrapper">
     <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=" alt="<%= caption %>" data-src="<%= image %>" />
     <figcaption class="photo__copy">
       <p class="photo__caption"><%= caption %></p>
     </figcaption>
   </figure>
+
+  <% if (reactions.enabled) { %>
+  <ul class="form-actions -inline photo_actions">
+    <% for (i = 0, count = reactions.data.length; i < count; i++) { %>
```

Comment:
![image](http://web-images.chacha.com/images/Gallery/7017/the-struggle-is-real-445124317-apr-17-2014-1-600x362.jpg)